### DISCUSSION TITLE
Possible to change design?

### DISCUSSION THREAD

**Discussion by jtbayly** [quarto-dev/quarto-cli#820] [Q&A] [answered]
I'm wondering if it's possible to do anything besides change the color scheme in an HTML book.

Thoughts?

**Comment by jjallaire** [comment, marked as answer]
you can create a quarto theme (https://quarto.org/docs/output-formats/html-themes.html) for your book which has a bunch of color variables (as you surmised) but can also set any of the nearly 1400 bootstrap SCSS variables that customize just about everything (https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss)

**Reply by jtbayly** [reply]
Thanks, I looked through that, but unless I'm missing something, that means the layout is fixed and can't be changed, right? I was hoping to recreate something a lot closer to the Gitbook theme used in Bookdown.

**Reply by jjallaire** [reply]
With SCSS and perhaps some JS you should be able to do quite a bit of customization. It's possible that's not enough though depending on what you have in mind.
